pdv/src/main/java/com/chuys/gshp/pdv/data/repository/ReportReportEntity.kt

```diff
@@ -0,0 +1,89 @@
+package com.chuys.gshp.pdv.data.repository
+
+import android.util.Log
+import com.chuys.gshp.pdv.domain.model.ReportReportModel
+import com.chuys.gshp.shared.data.database.sqlite.provider.SqliteDBProvider
+
+class ReportReportEntity {
+
+    private val sqliteHelper = SqliteDBProvider.SqliteDBHelper.getDbHelper()
+    private val tableName = "report_report"
+
+
+
+    fun writeReportReport(value: ReportReportModel) {
+        val db = sqliteHelper.writableDatabase
+
+        try {
+            var insStmnt = db.compileStatement(
+                "" +
```

Comment:
is necesary?

Reply:
no, it was a temporary change, due to the lack of knowledge that a report was already generated